oar/core/util.py

```
@@ -304,14 +318,15 @@ def is_payload_metadata_url_accessible(release: str) -> bool:
         requests.exceptions.RequestException: If any HTTP request fails
         subprocess.CalledProcessError: If oc command execution fails
     """
-    # get image pullspec from release stream 4-stable
+    # Get image pullspec from release stream (e.g., 4-stable, 5-stable)
     pullspec = None
-    url = f"https://amd64.ocp.releases.ci.openshift.org/api/v1/releasestream/4-stable/latest?prefix={release}"
+    major_version = get_major_version(release)
+    url = f"https://amd64.ocp.releases.ci.openshift.org/api/v1/releasestream/{major_version}-stable/latest?prefix={release}"
     resp = requests.get(url)
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)